compiler/include/symbol.h

```diff
@@ -237,6 +237,14 @@ class Symbol : public BaseAST {
   std::string deprecationMsg;
 
   const char* getDeprecationMsg() const;
+
+  // When printing the deprecation message to the console we typically
+  // want to filter out inline markup used for Sphinx (which is useful
+  // for when generating the docs). See:
+  // https://chapel-lang.org/docs/latest/tools/chpldoc/chpldoc.html#inline-markup-2
+  // for information on the markup.
```

Comment:
Personal preference - I like having comments on C function definitions rather than declarations, I think it's easier to remember to update them that way.